api/README.md

```diff
@@ -1,3 +1,61 @@
-# CompE+ api
+# CompE+ Backend
 
-TODO there should be an api specific README
+A REST API that supports the CompE+ [frontend](../www/README.md).
+
+# Setup
+
+Running `npm install` should download almost everything you need to get started.
+
+# Local Development
+
+To start a local development server run `npm run start:dev`. This will automatically pull in new changes as you update files.
+
+To test the code run `npm test`. You can also run `npm run test:watch` for an interactive test session.
+
+We also have a few scripts to keep the code quality up to snuff. Run `npm run format` to format all the code with Prettier and run `npm run lint` to make sure that ESLint doesn't complain about anything.
+
+The API makes use of a number of services e.g., database, blog storage, email. See the [dependencies](#Dependencies) section for more on how to locally emulate these services.
```

Comment:
idk if it's just the Github markdown renderer but the link isn't working but I tried doing it locally and it worked fine in vscode. i'm just gonna assume this link properly.

Reply:
The anchor link? It just scrolls to the dependencies section, it works for me when I open this branch on a separate page.

Reply:
Yeah works locally for me too

Reply:
ok ignore me haha